specifyweb/permissions/migrations/0004_intialize_defaults.py

```diff
@@ -10,6 +10,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('permissions', '0003_auto_20220321_1445'),
+        ('specify', '0002_geo'), # Fails without having collection updated
```

Comment:
Does this not cause inconsistent migration errors? You'll need to add to another a brand new migration, dependent on 0002_geo. Don't change any eisting migrations

Reply:
Ya, I'll add a new migration to prevent inconsistent migration errors for existing Specify instances.  The problem is that when running unit tests, the Django migrations start from scratch, and they fail at the permissions 0004 migration when using the collection model that has the updated field.

EDIT: I found a better solution to avoid this error by changing `for collection in Collection.objects.all():` to `for collection_id in Collection.objects.values_list('id'):`

Unfortunately, this kind of problem can occur when we change schema that existed before we had everything defined in Django migrations... if only migrations were recorded from the start, these problems are just the result of trying to rewrite history.

Reply:
I like the solution!